template.yml

```diff
@@ -0,0 +1,637 @@
+                  headers:
+                    Location:
+                      schema:
+                        type: string
+                      description: Relative path to the user URI
+              x-amazon-apigateway-integration:
+                uri:
+                  Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${UpdateUserHandler.Arn}/invocations
+                responses: {}
+                httpMethod: POST
+                type: aws_proxy
+          /institutions/{institution}/users:
+            get:
+              summary: Lists users of specified institution
+              parameters:
+                - in: path
+                  name: institution
+                  schema:
+                    type: string
+                  required: true
+              responses:
+                '200':
+                  description: List of users belonging in the specified institue
+                  content:
+                    application/json:
+                      schema:
+                        type: array
+                        items:
+                          $ref: '#/components/schemas/User'
+              security:
+                - CognitoUserPool: []
+              x-amazon-apigateway-integration:
+                uri:
+                  Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${ListByInstitutionHandler.Arn}/invocations
+                responses: {}
+                httpMethod: POST
+                type: aws_proxy
+        components:
+          schemas:
+            Role:
+              properties:
+                type:
+                  type: string
+                rolename:
+                  type: string
+            User:
+              properties:
+                type:
+                  type: string
+                username:
+                  type: string
+                institution:
+                  type: string
+                roles:
+                  type: array
+                  items:
+                    $ref: '#/components/schemas/Role'
+          securitySchemes:
+            CognitoUserPool:
+              type: apiKey
+              name: Authorization
+              in: header
+              x-amazon-apigateway-authtype: cognito_user_pools
+              x-amazon-apigateway-authorizer:
+                type: cognito_user_pools
+                providerARNs:
+                  - !Ref CognitoAuthorizerArn
+            ApiKey:
+              type: apiKey
+              name: Authorization
+              in: header
+              x-amazon-apigateway-authtype: custom
+              x-amazon-apigateway-authorizer:
+                type: request
+                authorizerUri:
+                  Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${LambdaAuthorizer.Arn}/invocations
+                identitySource: method.request.header.Authorization
+  UserRoleMappings:
```

Comment:
Move databasedefinition to NVA-infrastructure and handle dabasechanges separately.